middleware/checkUserCredentials.js

```diff
@@ -1,8 +1,8 @@
-const checkUserCredentials = (request, response) => {
+const checkUserCredentials = (request, response, next) => {
   const { username, password } = request.body;
 
   if (username === "lee" && password === "marsrover") {
-    response.redirect("/");
+    return next();
```

Comment:
You don't technically need to return anything here (but on an actual project you might consider keeping it if you're struggling to unit test this bit of code)

Reply:
Ah fair enough :) I think I'm in the habit of using `return next();` as this is a practice we use in our team, but I'm happy to remove the `return` here